simpleitemdecoration/src/main/java/com/dgreenhalgh/android/simpleitemdecoration/linear/DividerItemDecoration.java

```diff
@@ -105,8 +123,8 @@ private void drawHorizontalDividers(Canvas canvas, RecyclerView parent) {
      *               added
      */
     private void drawVerticalDividers(Canvas canvas, RecyclerView parent) {
-        int parentLeft = parent.getPaddingLeft();
-        int parentRight = parent.getWidth() - parent.getPaddingRight();
+        int parentLeft = mLeftInset + parent.getPaddingLeft();
+        int parentRight = mRightInset + parent.getWidth() - parent.getPaddingRight();
```

Comment:
I think that should be `parent.getWidth() - mRightInset - parent.getPaddingRight();`

Reply:
Upvote +1